compiler/test/stdlib/number.test.gr

```diff
@@ -300,17 +439,87 @@ assert 1 / Number.sign(0.0) == Float64.toNumber(Float64.infinity)
 // TODO(#693): Replace with -Infinity literal when it exists
 assert 1 / Number.sign(-0.0) == Number.neg(Float64.toNumber(Float64.infinity))
 
-// Number.sin
+// Number.sin - These test a range as we approximate sin
 assert Number.sin(0.0) == 0
 assert Number.sin(1) > 0.84147 && Number.sin(1) < 0.84148
 assert Number.sin(-1) < -0.84147 && Number.sin(-1) > -0.84148
 assert Number.sin(20) > 0.91294 && Number.sin(20) < 0.91295
+assert Number.sin(1/2) > 0.4794 && Number.sin(1/2) < 0.4795
+// TODO(#1478): Correct this test
+assert Number.sin(9223372036854775809) == 0.0
 assert Number.isNaN(Number.sin(Number.infinity))
 assert Number.isNaN(Number.sin(Number.nan))
 
-// Number.cos
+// Number.cos - These test a range as we approximate sin
 assert Number.cos(1) > 0.5403 && Number.cos(1) < 0.5404
 assert Number.cos(-1) > 0.5403 && Number.cos(-1) < 0.5404
 assert Number.cos(15) < -0.7596 && Number.cos(15) > -0.7597
+assert Number.cos(1/2) > 0.8775 && Number.cos(1/2) < 0.8776
+// TODO(#1478): Correct this test
```

Comment:
```suggestion
// TODO(#1478): Update this test when cos is unbounded
```

Reply:
Should i say `cos` or `sin`, `cos` and `tan` are only bounded because `sin` is bounded

Reply:
Ah, I think just `sin` then

Reply:
```suggestion
// TODO(#1478): Update this test when sin is unbounded
```